packages/react-components/react-tabs/src/components/Tab/useTabStyles.ts

```diff
@@ -203,7 +203,7 @@ const usePendingIndicatorStyles = makeStyles({
     ':before': {
       bottom: 0,
       ...shorthands.borderWidth(`calc(${tabIndicatorStrokeWidths.mediumHorizontal} / 2.0)`),
-      height: tabIndicatorStrokeWidths.mediumHorizontal,
+      height: 0,
```

Comment:
@smhigley - You mentioned possibly changing the box-siding to be content rather than border.  I investigate this and I need the border to be part of the height/width in order for the animation measurements to be accurate. I didn't find that one or the other made any difference once I set the width/height to zero.